### PR TITLE
feat(base): provide reset, fluid type, type scale, etc. - INNO-336

### PR DESCRIPTION
**Pull request by yhuard**
TODO:

- [x] Add type scales
- [x] Fix stylelint reported errors

### CODE REVIEW

**Comment by yhuard** [comment]
Updated with the font sizes 💪

File: framework/_global.scss [hunk 1]
```diff
@@ -2,17 +2,9 @@
 
 body {
   -moz-osx-font-smoothing: grayscale;
-  font-family: $font-family-base;
-  font-weight: 400;
-  line-height: 1.667;
+  -webkit-font-smoothing: antialiased;
```
Comment (emeryro) [inline framework/_global.scss]: We shouldn't force antialiasing and grayscale for default font (on Windows in doesn't change anything and it consumes resources).

File: framework/_global.scss [hunk 2]
```diff
@@ -26,29 +18,44 @@ body {
...
-  border: none;
+  border-width: 0;
 }
 
 .navbar-collapse,
 .navbar-fixed-top,
 .navbar {
-  border: none;
+  border-width: 0;
 }
```
Comment (emeryro) [inline framework/_global.scss]: navbar are related to Bootstrap and we don't use it, so we don't need to write specific rules

File: framework/_global.scss [hunk 3]
```diff
@@ -26,29 +18,44 @@ body {
...
+.sr-only {
+  border-width: 0;
+  clip: rect(0, 0, 0, 0);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+}
```
Comment (emeryro) [inline framework/_global.scss]: sr-only class should be put in utilities

File: framework/base/_fluid-type.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
+// Implement fluid type
+// More information: https://websemantics.uk/tools/responsive-font-calculator/
+
+// Default 1rem(16px) @ 30rem(480px) increasing to 1.5rem(24px) @ 120rem(1920px)
+$fluid-type-font-size-min: map-get($font-size, 'm') !default;
+$fluid-type-font-size-max: map-get($font-size, 'xl') !default;
```
Comment (emeryro) [inline framework/base/_fluid-type.scss]: To be checked on a large screen if font isn't too big

File: framework/_global.scss (hunk 2 above)
Comment (yhuard) [inline framework/_global.scss]: Time to break everything and to remove bootstrap!

File: framework/base/_fluid-type.scss (hunk 4 above)
Comment (yhuard) [inline framework/base/_fluid-type.scss]: Indeed

**Pull request merged**